vendor/github.com/coreos/etcd/raft/raft.go

```diff
@@ -135,6 +156,23 @@ type Config struct {
 	// steps down when quorum is not active for an electionTimeout.
 	CheckQuorum bool
 
+	// PreVote enables the Pre-Vote algorithm described in raft thesis section
+	// 9.6. This prevents disruption when a node that has been partitioned away
+	// rejoins the cluster.
+	PreVote bool
```

Comment:
I'd imagine we want to leave this off unless we can confirm that all managers support it.